domain-server/resources/web/web-new/src/components/dialogs/ConnectMetaverse.vue

```diff
@@ -0,0 +1,131 @@
+<!--
+//  ConnectMetaverse.vue
+//
+//  Created by Kalila L. on Sep. 5th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+
+<template>
+    <q-card-section>
+        <div class="row no-wrap items-center">
+            <div class="col text-h4 ellipsis">
+                Metaverse
+            </div>
+        </div>
+
+    </q-card-section>
+
+    <q-separator />
+
+    <q-card-section>
+        <MetaverseLogin @loginResult="onLoginAttempt"></MetaverseLogin>
+    </q-card-section>
+</template>
+
+<script>
+// FIXME: Needs to be done correctly. Also universally? Maybe window.axios?
+const axios = require("axios");
+
+// Components
+import MetaverseLogin from "../components/login/MetaverseLogin.vue";
+// Modules
+import Log from "../../modules/utilities/log";
+
+export default {
+    name: "ConnectMetaverse",
+
+    components: {
+        MetaverseLogin
+    },
+
+    emits: ["connectionResult"],
+
+    data: () => ({
+    }),
+
+    methods: {
+        onLoginAttempt (result) {
```

Comment:
Not really, there is no await for the function itself, only Axios' function. The reason we don't need it is because the function goes on autopilot from there and isn't meant to return anything directly, it self handles and emits results to its parent.